### PR TITLE
Discord Emote to Unicode

### PR DESCRIPTION
**Pull request by levi506**
/N{} does not work with The Boxed Numbers and Heart

### CODE REVIEW

File: cogs/economy.py [hunk 1]
```diff
@@ -12,19 +12,32 @@
 import time
 import logging
 
+slot_emotes = {
+	'TWO': '2\u20e3',
```
Comment (Chovin) [inline cogs/economy.py]: probably won't compile. mixed indentation, tabs/spaces

File: cogs/economy.py [hunk 2]
```diff
@@ -12,19 +12,32 @@
...
 slot_payouts = """Slot machine payouts:
-    :two: :two: :six: Bet * 5000
-    :four_leaf_clover: :four_leaf_clover: :four_leaf_clover: +1000
-    :cherries: :cherries: :cherries: +800
-    :two: :six: Bet * 4
-    :cherries: :cherries: Bet * 3
+    {TWO}{TWO}{SIX}Bet * 5000
+    {FOUR LEAF CLOVER} {FOUR LEAF CLOVER} {FOUR LEAF CLOVER} +1000
+    {CHERRIES} {CHERRIES} {CHERRIES} +800
+     {TWO}{SIX}Bet * 4
```
Comment (Chovin) [inline cogs/economy.py]: were the missing spaces here and above on purpose?

File: cogs/economy.py [hunk 3]
```diff
@@ -479,11 +492,11 @@ def already_in_list(self, accounts, user):
             await self.bot.say("{0} You need an account with enough funds to play the slot machine.".format(author.mention))
 
     async def slot_machine(self, message, bid):
-        reel_pattern = [":cherries:", ":cookie:", ":two:", ":four_leaf_clover:",
-                        ":cyclone:", ":sunflower:", ":six:", ":mushroom:", ":heart:", ":snowflake:"]
+        reel_pattern = ["{CHERRIES}".format(**slot_emotes), "{COOKIE}".format(**slot_emotes), "{TWO}".format(**slot_emotes), "{FOUR LEAF CLOVER}".format(**slot_emotes),
```
Comment (Chovin) [inline cogs/economy.py]: please find a more concise way to do this

File: cogs/economy.py [hunk 4]
```diff
@@ -505,11 +505,11 @@ def already_in_list(self, accounts, user):
...
-            reels[0][0] + " " + reels[1][0] + " " + reels[2][0] + "\n"
-        display_reels += ">" + reels[0][1] + " " + \
-            reels[1][1] + " " + reels[2][1] + "\n"
-        display_reels += "  " + reels[0][2] + " " + \
-            reels[1][2] + " " + reels[2][2] + "\n"
+            reels[0][0].format(**slot_emotes) + " " + reels[1][0].format(**slot_emotes) + " " + reels[2][0].format(**slot_emotes) + "\n"
+        display_reels += ">" + reels[0][1].format(**slot_emotes) + " " + \
+            reels[1][1].format(**slot_emotes) + " " + reels[2][1].format(**slot_emotes) + "\n"
+        display_reels += "  " + reels[0][2].format(**slot_emotes) + " " + \
+            reels[1][2].format(**slot_emotes) + " " + reels[2][2].format(**slot_emotes) + "\n"
```
Comment (Chovin) [inline cogs/economy.py]: suggest formatting it all afterwards instead of at each concat

File: cogs/economy.py [hunk 5]
```diff
@@ -505,11 +505,11 @@ def already_in_list(self, accounts, user):
...
 
         if line[0] == "{TWO}".format(**slot_emotes) and line[1] == "{TWO}".format(**slot_emotes) and line[2] == "{SIX}".format(**slot_emotes):
```
Comment (Chovin) [inline cogs/economy.py]: 1. you only really need the values for printing
2. you'll probably never win since `line` only has the emote keys, not the values

**Review by Chovin — CHANGES_REQUESTED** [review]
please see comments

File: cogs/economy.py [hunk 6]
```diff
@@ -13,11 +13,11 @@
 import logging
 
 slot_emotes = {
-    'TWO': '2\u20e3',
-    'SIX': '6\u20e3',
+    'TWO': '"2\u20e3"',
```
Comment (Chovin) [inline cogs/economy.py]: huh?

File: cogs/economy.py [hunk 7]
```diff
@@ -511,27 +511,27 @@ def already_in_list(self, accounts, user):
         display_reels += "  " + reels[0][2].format(**slot_emotes) + " " + \
             reels[1][2].format(**slot_emotes) + " " + reels[2][2].format(**slot_emotes) + "\n"
 
-        if line[0] == "{TWO}".format(**slot_emotes) and line[1] == "{TWO}".format(**slot_emotes) and line[2] == "{SIX}".format(**slot_emotes):
+        if line[0] == "2\u20e3" and line[1] == "2\u20e3" and line[2] == "6\u20e3":
```
Comment (Chovin) [inline cogs/economy.py]: I think you're kinda going backwards here

**Review by Chovin — CHANGES_REQUESTED** [review]
please reread previous comments

**Comment by Chovin** [comment]
most recent changes seem acceptable for the scope of this PR
Nice job :)
just need to test it now to make sure it works for sure. sadly I don't have the time to do this.
edit: wait.. you just made a change.. time to rereview

**Comment by Chovin** [comment]
ah. 👍  from me once this gets tested

**Comment by Twentysix26** [comment]
Closing in favor of #538 👍

**Pull request closed without merging**